modules/monitoring-getting-detailed-information-about-a-target.adoc

```diff
@@ -0,0 +1,38 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-metrics-targets.adoc
+
+[id="getting-detailed-information-about-a-target_{context}"]
+= Getting detailed information about a target
+
+On the **Target details** page, you can view detailed information about a metric target.
+
+.Prerequisites
+
+* You have access to the cluster as an administrator for the project for which you want to view metrics targets.
```

Comment:
Why is this bold? I'm not used to seeing an entire sentence bold.

Reply:
The bold formatting here is to be consistent with the parallel topics "Managing Metrics" and "Managing Alerts," both of which use this formatting.